.github/workflows/gpu-parity-pocl.yml

```
@@ -0,0 +1,99 @@
+#   * AIDOTNET_OPENCL_ALLOW_CPU=1  — lets the OpenCL backend bind a CPU device as a fallback
+#                                    when no GPU is present (default off; GPU always preferred,
+#                                    so this changes nothing on real-GPU machines).
+#   * AIDOTNET_REQUIRE_GPU_TESTS=1 — turns "no GPU device" from a silent Skip into a hard
+#                                    failure, so a broken POCL setup fails loudly here instead
+#                                    of quietly skipping the whole gate.
+#
+# Not yet a required check. Promote it in branch protection only after the first green run
+# confirms POCL compiles + runs the kernel cache in CI; until then it is a live, diagnostic
+# gate whose job status reflects real CPU-vs-GPU divergence.
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)